frontend/server/src/DAO/QualityNominations.php

```diff
@@ -123,6 +123,73 @@ public static function reviewerHasQualityTagNominatedProblem(
         ) > 0;
     }
 
+    /**
+     * Returns the contents and id of a nomination for a given problem and user.
+     */
+    public static function getReviewedData(
+        \OmegaUp\DAO\VO\Identities $identity,
+        \OmegaUp\DAO\VO\Problems $problem
+    ): array {
+        // This SQL is basically the same as the one in QualityNominations::getNominationStatusForProblem
+        // Is there a way to merge them?
+
+        // Count is not neccesary, because we can just check if the query returned a non null value
+        $sql = "
+            SELECT
+                qn.qualitynomination_id,
+                qn.contents
+            FROM
+                QualityNominations qn
+            INNER JOIN
+                Identities i ON i.user_id = qn.user_id
+            WHERE
+                nomination = 'quality_tag' AND
+                i.identity_id = ? AND
+                qn.problem_id = ?";
+
+        /** @var array{contents: string, qualitynomination_id: int}|null */
+        $query = \OmegaUp\MySQLConnection::getInstance()->GetRow(
+            $sql,
+            [$identity->identity_id, $problem->problem_id]
+        );
+
+        if (is_null($query)) {
+            return [];
+        }
+
+        $contents = json_decode($query['contents'], true);
```

Comment:
Aquí como psalm no sabe que es lo que se está decodificando, es necesario agregar la anotación del tipo explícitamente:
```suggestion
        /** @var  array{quality_seal: bool, tag: string} $contents */
        $contents = json_decode($query['contents'], true);
```
eso es lo que debería regresar, según lo indicaste en la anotación de `QualityNominationContents` que agregaste en el controlador.